xapian-applications/omega/templates/emptydocs

```diff
@@ -1,6 +1,11 @@
+<!DOCTYPE html>
+<html>
```

Comment:
You cannot omit the `head` element in html5 if it's empty. It's easiest to put one in with a `title` matching the `h1`.